tests/trestle/utils/list_utils_test.py

```diff
@@ -81,3 +84,39 @@ def test_get_item_from_list() -> None:
     item = list_utils.get_item_from_list(items, 4, lambda x: list(x.values())[0])
     assert item == {'b': 4}
     assert items == [{'a': 2}, {'b': 4}, {'c': 6}]
+
+
+def test_comma_sep_to_list() -> None:
+    """Test comma sep to list."""
+    assert list_utils.comma_sep_to_list(None) == []
+    assert list_utils.comma_sep_to_list('a') == ['a']
+    assert list_utils.comma_sep_to_list('') == ['']
```

Comment:
Why None and empty string give different results? Should it be the same?

Reply:
It's a corner case, but this has some value in distinguishing an empty list from a list containing a single empty string.

Reply:
Is that indeed needed? It complicates logic a bit as it requires the user of the function to handle two cases

Reply:
I'm fine making them the same

Reply:
they now behave the same